docs/app/pages/components/sections/panels/item-display-panel/item-display-panel.component.html

```diff
@@ -103,9 +97,9 @@
 
 <hr>
 
-<p>To add the main body of content to the item display panel the <code>ux-item-display-panel-content</code> directive can be added allowing you to add any required content below the header.</p>
+<p>To add the main body of content to the item display panel the <code>uxItemDisplayPanelContent</code> directive can be added allowing you to add any required content below the header.</p>
```

Comment:
Should probably mention that it should be added as an attribute for both of these directives